src/olympia/lib/settings_base.py

```diff
@@ -706,153 +706,44 @@ def get_db_config(environ_var, atomic_requests=True):
         # js/node_lib/* files are copied in Makefile-docker - keep both lists in sync
         'common': (
             'js/node_lib/underscore.js',
-            'js/zamboni/browser.js',
-            'js/amo2009/addons.js',
             'js/zamboni/init.js',
-            'js/impala/capabilities.js',
+            'js/zamboni/capabilities.js',
             'js/lib/format.js',
             'js/node_lib/jquery.cookie.js',
             'js/zamboni/storage.js',
-            'js/zamboni/buttons.js',
-            'js/zamboni/tabs.js',
             'js/common/keys.js',
-            # jQuery UI
-            'js/node_lib/ui/version.js',
-            'js/node_lib/ui/data.js',
-            'js/node_lib/ui/disable-selection.js',
-            'js/node_lib/ui/ie.js',
-            'js/node_lib/ui/keycode.js',
-            'js/node_lib/ui/labels.js',
-            'js/node_lib/ui/jquery-patch.js',
-            'js/node_lib/ui/plugin.js',
-            'js/node_lib/ui/safe-active-element.js',
-            'js/node_lib/ui/safe-blur.js',
-            'js/node_lib/ui/scroll-parent.js',
-            'js/node_lib/ui/focusable.js',
-            'js/node_lib/ui/tabbable.js',
-            'js/node_lib/ui/unique-id.js',
-            'js/node_lib/ui/position.js',
-            'js/node_lib/ui/widget.js',
-            'js/node_lib/ui/menu.js',
-            'js/node_lib/ui/mouse.js',
-            'js/node_lib/ui/autocomplete.js',
-            'js/node_lib/ui/sortable.js',
             'js/zamboni/helpers.js',
-            'js/common/banners.js',
             'js/zamboni/global.js',
-            'js/amo2009/global.js',
-            'js/common/ratingwidget.js',
-            'js/node_lib/jqModal.js',
             'js/zamboni/l10n.js',
-            'js/zamboni/debouncer.js',
-            # Homepage
-            'js/zamboni/homepage.js',
```

Comment:
I don't see that file being removed in this PR. Am I missiing something?

Reply:
I need to go back to check, it looks like there are a few files I forgot to remove in zamboni/. That's why this PR is a draft :) (It's also broken right now, need to tweak things out a bit)

Reply:
https://github.com/mozilla/addons-server/pull/18755/commits/f1e136a1ea0fb7242597aede0aed79325c6a1bb2